workbench-core/example/infrastructure/integration-tests/tests/datasets/byob.test.ts

```diff
@@ -0,0 +1,126 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AwsService } from '@aws/workbench-core-base';
```

Comment:
This line is failing the unit tests. (`AwsService` not used in the code causing es-lint warning)